packages/themes/src/base.css

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
Not sure if we should include this (or base styles at all)

Reply:
Base styles could be useful, but it would be nice if we hid the extra imports from the users.